ui/packages/catalog-ui-search/src/main/webapp/component/search-form-interactions/search-form-interactions.view.js

```diff
@@ -41,106 +43,110 @@ module.exports = Marionette.ItemView.extend({
   },
   ui: {},
   initialize: function() {
-    this.listenTo(
-      user.getQuerySettings(),
-      'change',
-      this.checkIfDefaultSearchForm
-    )
+    this.listenTo(user.getQuerySettings(), 'change', function() {
+      this.$el.toggleClass(
+        'is-current-template',
+        user.getQuerySettings().isTemplate(this.model)
+      )
+    })
   },
   onRender: function() {
-    this.checkIfSubscribed()
-    this.checkIfDefaultSearchForm()
-    this.checkIfResultForm()
-    this.isSystemTemplate()
-    this.isShareableTemplate()
-  },
-  checkIfSubscribed: function() {
     this.$el.toggleClass('is-subscribed', Boolean(this.model.get('subscribed')))
+    this.$el.toggleClass(
```

Comment:
✏️ I think using the old `checkIf*` and `is*Template` methods was more readable than inlining them here.